src/lib/crypt.test.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * @jest-environment node
+ */
+
+import { hash } from './crypt';
+
+describe('Crypt', () => {
+  describe('hash()', () => {
+    it('should generate hash for the string', async () => {
+      const str = 'test';
+      const result = await hash(str);
```

Comment:
Minor rename suggestion to avoid using too generic variable names such as `data` or `results`:

```suggestion
      const hash = await hash(str);
```

or 

```suggestion
      const generatedHash = await hash(str);
```

Reply:
Fixed in https://github.com/nimblehq/ic-nextjs-prabesh-dahal/pull/44/commits/772649044e477fe0425328fbaae9c5acd0eccb73